niworkflows/reports/core.py

```diff
@@ -102,17 +102,19 @@ class Reportlet(Element):
     True
 
     >>> r = Reportlet(bl, out_figs, config={
-    ...     'title': 'Some Title', 'bids': {'datatype': 'anat', 'space': '.*'},
+    ...     'title': 'Some Title',
+    ...     'bids': {'datatype': 'anat', 'space': '.*', 'regex_search': True},
```

Comment:
Not a huge fan of specifying this in the config, but I couldn't figure out another way to make this work without breaking other things.